packages/astro/src/assets/fonts/utils.ts

```diff
@@ -251,7 +263,8 @@ export async function resolveFontFamily({
 		...family,
 		nameWithHash,
 		provider: await resolveProvider({
```

Comment:
nit: It's a bit inconsistent that we have a function `resolveProvider` that uses `resolveFontFamily`, but now we have this very function - `resolveFontFamily` - that uses `resolveEntrypoint` **directly**.

Wouldn't it make more sense to have a function called `resolveVariantSources`, which uses `resolveEntrypoint` under the hood?

Reply:
Yeah fine by me!